x-pack/plugins/secops/public/components/event_details/json_view.tsx

```diff
@@ -39,3 +40,6 @@ export const JsonView = pure<Props>(({ data }) => (
     }
   </JsonEditor>
 ));
+
+const buildBackJsonView = (data: DetailItem[]) =>
```

Comment:
what does `Back` in the name imply?

Reply:
Nothing, because I bringing `json back`. Call Justin!!!

Good point, I will remove `back`